pilot/pkg/proxy/envoy/v2/eds.go

```diff
@@ -133,8 +140,17 @@ func networkEndpointToEnvoyEndpoint(e *model.NetworkEndpoint) (*endpoint.LbEndpo
 	if err != nil {
 		return nil, err
 	}
+
 	addr := util.GetNetworkEndpointAddress(e)
+
+	epWeight := e.LbWeight
```

Comment:
we may want to clean up this code duplication in the future, it makes it easy to make a mistake when making a change like ths